src/styles/blocks/header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+}
+
+.nav {
```

Comment:
Checklist p.9 [BEM] - create a separate file per each BEM block styles that have the same name as the block.